testkitbackend/backend.py

```diff
@@ -0,0 +1,63 @@
+from json import loads, dumps
+from inspect import getmembers, isfunction
+import testkitbackend.requests as requests
+
+
+# Collect all request handlers
+requestHandlers = dict([m for m in getmembers(requests, isfunction)])
```

Comment:
I think it could be a good idea or inject this information on the constructor as param (so, the Server will create the object and pass to Backend constructor) or it be created on the Backend constructor (which is a bit less performatic). Global variables could have strange behaviours if some one change it value by 'accident'

Reply:
I move it into the backend, good point!